UnitTesting/SpecificLeaksEventsTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WaterLog_Backend;
+using WaterLog_Backend.Controllers;
+using WaterLog_Backend.Models;
+using WebApplication1;
+
+namespace UnitTesting
+{
+    class SpecificLeaksEventsTest
+    {
+       
+        private IQueryable<SegmentLeaksEntry> mockData = new List<SegmentLeaksEntry>
+        {
+            new SegmentLeaksEntry
+            {
+                Id = 1, SegmentsId = 1, Severity = "High", ResolvedStatus = "Unresolved",
+                LatestTimeStamp = DateTime.Now, OriginalTimeStamp = DateTime.Today.Date
+            },
+
+            new SegmentLeaksEntry
+            {
+                Id = 2, SegmentsId = 1, Severity = "Low", ResolvedStatus = "Resolved",
+                LatestTimeStamp = DateTime.Now, OriginalTimeStamp = DateTime.Today.Date
+            }
+        }.AsQueryable();
+        
+        [TestCase(1)]
+        public async Task getvalue(int id) {
```

Comment:
Method name not conforming to naming standards. Remember to use Pascal Casing.

Reply:
Fixed